### ISSUE TITLE
Double modules in admin view?

### ISSUE THREAD

**Issue by JoachimTorngren** [Billmate/opencart#178]
Opencart 2.3.0.2 / Billmate 2.2.3

Having Billmate files in both...
admin\controller\payment
admin\controller\extension\payment
... makes the modules displayed twice in admin (extensions/payments)

**Comment by Boxedsolutions** [comment]
Yes we are aware of that. This is due to opencarts stupid backward compatibility fix that messes things up for us that wants one module for everything.

**Comment by JoachimTorngren** [comment]
OK. Perhaps just inform in README about the issue? As a non-tech guy I get a bit suspicious when an important core module behaves like that.

**Comment by Boxedsolutions** [comment]
Of course we will add this to the next release. Thanks for your feedback
